Application/Client/src/Employee.cpp

```diff
@@ -0,0 +1,359 @@
+#include <iostream>
+#include <iomanip>
+#include <limits>
+
+#include "Employee.h"
+#include "Utils.h"
+#include "ClientSocket.h"
+#include "RequestType.h"
+
+enum class EmployeeOptions {
+    DISPLAY_MENU,
+    VIEW_SPECIFIC_DATE_MENU,
+    UPDATE_PROFILE,
+    VIEW_PROFILE_DETAILS,
+    GIVE_VOTE,
+    GIVE_FEEDBACK,
+    LOGOUT
+};
+
+Employee::Employee(std::string username) : logged_username(username) {
+}
+
+void Employee::employeeLandingPage() {

```

Comment:
The landing page functions in all these classes do not seem to belong to these classes. The employee, chef etc. do not have a behaviour of landing page. That could be handled elsewhere.